src/components/Pages/HomePage.js

```diff
@@ -26,10 +27,17 @@ export const HomePage = () => {
   }, []);
 
   return (
-    <div>
+    <div className="home-page">
       <SeriesSearchForm />
       <div className="tv-show-poster-section">
-        {_.map(episodesData, (data, index) => (
+        {tvPosterData && !tvPosterData.length && (
+          <React.Fragment>
+            {[...Array(5)].map((item, index) => {
```

Comment:
5 is a fairly arbitrary number. it actually has some semantic meaning. probably make a constant. like `const PREVIEW_SERIES_COUNT = 5` or something. then on line 15, switch the 5 with the variable